data/settings.py

```diff
@@ -10,7 +10,7 @@
 # See https://docs.djangoproject.com/en/1.9/howto/deployment/checklist/
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = bool(os.getenv("DEBUG", False))
+DEBUG = True if (os.getenv('DEBUG') == 'true') else False
```

Comment:
ah, just to answer your question - it won't work :-) And I think that's alright, env vars are strings not python types :-)

Still +1 on ``django-environ``, it is lovely